README.md

```diff
@@ -119,8 +118,7 @@ The application accepts several flags:
 - `--path-prefix`: path prefix for the application (default is `/`). This
   can be useful when running the application behind a reverse proxy.
 - `--provider`: cloud provider to use (default is `aws`).
-- `--resource-group-name`: name of the resource group to use (required for
-  Azure).
+- `--resource-group-name`: name of the resource group to use for azure (default queried from subscription ID).
```

Comment:
This comment doesn't indicate what the flag does. Why would I pass this if it's queried from the subscription ID?